modules/monitoring/variables.tf

```diff
@@ -119,3 +119,53 @@ variable "route53_availability_check_region" {
   description = "Define route53 health check region"
   type        = string
 }
+
+
+#KMS CMK VARS:
+
+variable "tags" {
+  description = "A map of tags to assign to the resources."
+  type        = map(string)
+  default     = {}
+}
+
+
+variable "kms_master_key_id" {
+  description = "ARN of the Customer Managed Key (CMK)."
+  type        = string
+  default     = ""
+}
+
+variable "cmk_description" {
+  default = "KMS Key to encrypt SNS"
+}
+
+variable "key_spec" {
+  default = "SYMMETRIC_DEFAULT"
+}
+
+variable "enabled" {
```

Comment:
can we switch the name to something that is giving more information what will be enabled?

Reply:
renamed to key_enabled and added description